Tools/MGCB/BuildContent.cs

```diff
@@ -430,17 +435,25 @@ public void Build(out int successCount, out int errorCount)
                     // Only copy if the source file is newer than the destination.
                     // We may want to provide an option for overriding this, but for
                     // nearly all cases this is the desired behavior.
-                    if (File.Exists(dest))
+                    if (File.Exists(dest) && !Rebuild)
                     {
                         var srcTime = File.GetLastWriteTimeUtc(c.SourceFile);
                         var dstTime = File.GetLastWriteTimeUtc(dest);
                         if (srcTime <= dstTime)
                         {
-                            Console.WriteLine("Skipping {0}", c.SourceFile);
+                            if (string.IsNullOrEmpty(c.Link))
+                                Console.WriteLine("Skipping {0}", c.SourceFile);
+                            else
+                                Console.WriteLine("Skipping {0} => {1}", c.SourceFile, c.Link);
```

Comment:
@cra0zy @nkast @Jjagg 

Since the change to support copy items with a link...  it seemed appropriate to show this by showing the link.  How i have it here the logging looks like:
```
C:\Souce\File\Input\File.png => Dest\Folder\File.png
```
The `=>` seemed like good shorthand to mean a linked copy.

Thoughts?

Reply:
Agreed, nice idea.